official/resnet/resnet_run_loop.py

```diff
@@ -26,21 +26,27 @@
 import argparse
 import os
 
-import tensorflow as tf  # pylint: disable=g-bad-import-order
+# pylint: disable=g-bad-import-order
+import tensorflow as tf
+from tensorflow.contrib.distribute.python import mirrored_strategy
```

Comment:
actually you shouldn't need this. you can directly use them as 
`tf.contrib.distribute.MirroredStrategy`
`tf.contrib.distribute.OneDeviceStrategy`